plugins/ltac2/g_ltac2.mlg

```diff
@@ -933,10 +933,14 @@ open Vernacextend
 }
 
 VERNAC { tac2mode } EXTEND VernacLtac2
-| ![proof] [ ltac2_expr(t) ltac_use_default(with_end_tac) ] =>
-    { classify_as_proofstep } -> {
-(*     let g = Option.default (Proof_global.get_default_goal_selector ()) g in *)
-  fun ~pstate -> Tac2entries.call ~pstate ~with_end_tac t }
+| ![proof] [ ltac_selector_opt(g) ltac2_expr(t) ltac_use_default(with_end_tac) ] =>
```

Comment:
I've just realized that the `ltac_selector` argument is defined in `G_ltac`, which has no mli file. Probably not an issue that should be solved in this PR but we should definitely add interfaces to all mlg files so that we know what kind of semi-automatic crap we may depend on.

Reply:
https://github.com/coq/coq/pull/10273

Reply:
Pffft...